lnrpc/walletrpc/walletkit.proto

```diff
@@ -1166,11 +1166,19 @@ message BumpFeeRequest {
     with.
     */
     uint64 sat_per_vbyte = 5;
+
+    // Indicates whether to include the raw transaction hex for
```

Comment:
Need to mention that this will wait for the batcher to create a transaction, which might take a while, at least the duration of the configured `sweeper.batchwindowduration` value (default is 30 seconds).